### PR TITLE
Create LICENSE

### LINKED ISSUE #8: License
I noticed this repo and mtasa-resources don't have a license. I was looking to add a file from AMXModX, which is GPL 3.0. (namely a slightly old version of format.cpp, which is used by SA-MP to format strings in custom versions of the `printf` and `format`. The natives provided by `amx_ConsoleInit` are not actually used). mtasa-blue is GPL 3.0, so my assumption is that everything here also is GPL 3.0, but I'm not 100% if that's the case. It would be ideal to specify the license ASAP to avoid any problems if we want to use external stuff.

On the other hand, AMX/Pawn uses the "zLib/libpng" license which is very permissive (and therefore GPL compatible), so no problems on that regard.

As a side note, I realized SA-MP violates GPL because it doesn't publicly provide the source of this file!

### PR DESCRIPTION
**Pull request by qaisjp**
This PR fixes #8 by adding the same license that the publicsdk uses: https://github.com/multitheftauto/mtasa-blue/blob/master/Shared/publicsdk/LICENSE?rgh-link-date=2020-06-07T13%3A31%3A59Z

These terms match the license described in the original install manual: https://github.com/multitheftauto/amx/blob/bccc107e89d8b605463d2ee86bbdf4fbe50c2897/amx-deps/install.html#L30-L38

### CODE REVIEW

**Review by theSpool — APPROVED** [review]

**Comment by qaisjp** [comment]
I have received approval from `arc_` for this PR, by email. Waiting for response by adamix.

**Review by colistro123 — APPROVED** [review]

**Comment by Disinterpreter** [comment]
> 
> 
> I have received approval from `arc_` for this PR, by email. Waiting for response by adamix.

I'm sorry, but you never get the answer from adamix.

https://forum.mtasa.com/topic/93686-%D0%BF%D0%B0%D0%BC%D1%8F%D1%82%D1%8C-%D0%BE-adamixe/

https://translate.google.com/translate?hl=&sl=ru&tl=en&u=https%3A%2F%2Fforum.mtasa.com%2Ftopic%2F93686-%D0%BF%D0%B0%D0%BC%D1%8F%D1%82%D1%8C-%D0%BE-adamixe%2F <- google translate

**Review by ghost — APPROVED** [review]

**Pull request merged**
